README.md

```diff
@@ -121,6 +120,54 @@ bfr = BasisFunctionRegressor(blr, ϕ)
 var(bfr(X)) == var(blr(ϕ(X)))
 ```
 
+## Drawing Function Samples
+
+There are two ways of drawing samples from `f::Union{BayesianLinearRegressor,BasisFunctionRegressor}`. The first is by using the `AbstractGPs` API (as in the above examples) where a `FiniteBLR` projection is first produced at a fixed set of input locations `X` with some specified observation noise `Σy`: `fx = f(X, Σy)::FiniteBLR`. Then, observations (including noise) at `X` can be sampled directly with `rand(rng, fx)`.
+The other way is to draw a sample of an entire function from `f` using `g = rand(rng, f)`. This samples a value for the weights `w ~ N(mw, Λw)` and produces a function `g(X) = w'X` which can be evaluated at any input locations. Note that this method corresponds to drawing samples of noiseless observations.
+
+``` julia
+using AbstractGPs, BayesianLinearRegressors, LinearAlgebra, Random, Plots
+
+# Fix seed for re-producibility.
+rng = MersenneTwister(123456)
+
+X = RowVecs(hcat(range(-1.0, 1.0, length=5)))
+f = BayesianLinearRegressor(zeros(2), Diagonal(ones(2)))
+
+## The first method of drawing samples - using the AbstractGPs API:
+# Index into the regressor at fixed inputs X and assume homoscedastic observation noise `Σ_noise`.
+N = 10
+X = ColVecs(hcat(range(-5.0, 5.0, length=N), ones(N))')
+Σ_noise = exp(randn())
```

Comment:
Maybe just fix this to a known scalar to the make the example trivial to reproduce?
```suggestion
Σ_noise = 0.1
```